Remotion/Data/DomainObjects.Validation.UnitTests/DomainObjectAttributesBasedValidationPropertyRuleReflectorTests/DomainObjectMixin_DOABCPRRTest.cs

```diff
@@ -435,28 +417,51 @@ public void GetRemovablePropertyValidators_MandatoryStringPropertyAndValidationM
     [Test]
     public void GetNonRemovablePropertyValidators_NullableStringPropertyAttribute ()
     {
+      var lengthValidationMessageStub = new Mock<ValidationMessage>();
+      _validationMessageFactoryStub
+          .Setup(
+              _ => _.CreateValidationMessageForPropertyValidator(
+                  It.IsAny<MaximumLengthValidator>(),
+                  PropertyInfoAdapter.Create(_mixinPropertyWithNullableStringPropertyAttribute)))
+          .Returns(lengthValidationMessageStub.Object);
+
       var result = _propertyWithNullableStringPropertyAttributeReflector.GetNonRemovablePropertyValidators().ToArray();
 
-      Assert.That(result.Any(), Is.False);
```

Comment:
No, I can't tell you why it was written that way. Either because it wasn't possible back than or because it wasn't noticed during review.